android/java/res/values/brave_colors.xml

```diff
@@ -214,7 +214,7 @@
 
     <color name="rewards_panel_bg_color">#FAFCFF</color>
     <color name="rewards_panel_text_color">#212529</color>
-    <color name="rewards_panel_secondary_text_color">#868E96</color>
+    <color name="rewards_panel_secondary_text_color">#495057</color>
```

Comment:
This color is used in many places. Like in tipping panel. May be better to create new color.